### ISSUE TITLE
Only one bounty is detected if multiple fixes are mentioned in the PR comment

### ISSUE THREAD

**Issue by annadanchenko** [status-im/commiteth#132]
[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

One PR can potentially fix several issues. So developer can write comment "Fixes #XX and Fixes #YY" where XX and YY are issue numbers. Currently, only XX will be detected as bounty. Would be good to detect all issues in the comment and pay multiple bounties.

#### Expected behavior
[comment]: # (Describe what you expected to happen.)
There are 2 (or X) open issues with bounties. PR with a comment " #XX and Fixes #YY" is submitted. 
As soon as PR is submitted:
1. For Owner on the SOB website, in Manage Payouts there are 2 (or X) open issues shown in New claims 
As soon as PR is merged:
1. Comment on each mentioned GH issue changes to "Status: pending maintainer confirmation
Winner: username" 
2. For Owner on the SOB website, in Manage Payouts, for all mentioned issues Confirm button is active, so payout can be confirmed

#### Actual behavior
[comment]: # (Describe what actually happened.)
Only first issue in the PR comment is detected as bounty and payout can be confirmed only for this first issue

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

- Create 2 or X issues with bounties
- Submit PR with comment "Fixes #XX and Fixes #YY" (possibly mo Fixes #NN if you have X issues)
- as anyone: check SOB site in Activity feed that claims are submitted for issues #XX and #YY
- as an owner: Check SOB site in Manage payouts to see that claims are submitted for issues #XX and #YY
- Merge PR
- as an owner: Check SOB site in Manage payouts to see that claims can be Confirmed for issues #XX and #YY
- Confirm claims and check that author gets bounties

**Comment by rcullito** [comment]
This seems like it would be a large investment of developer effort, so we would want to find out how often this is happening in practice before starting in. Going to create a label called `needs metrics` for this class of issue that would be nice to have but would depend on frequency.
